packages/gg_images/PKGBUILD

```diff
@@ -0,0 +1,37 @@
+pkgname=gg_images
+pkgver=20.fe35153
+pkgrel=1
+pkgdesc='GG profile photo downloader'
+arch=('x86_64')
+groups=('blackarch')
+url='https://codeberg.org/nanoory/GG_Images'
+license=('GPL-3.0')
+makedepends=('git' 'go-pie')
+source=("$pkgname::git+https://codeberg.org/nanoory/GG_Images")
+sha512sums=('SKIP')
+
+pkgver() {
+	cd $pkgname
+
+	echo $(git rev-list --count HEAD).$(git rev-parse --short HEAD)
+}
+
+build() {
+  cd $pkgname
+
+  GOPATH="$srcdir" go mod download
+  GOPATH="$srcdir" go build \
+    -trimpath \
+    -buildmode=pie \
+    -mod=readonly \
+    -modcacherw \
+    -ldflags "-s -w" \
+    -o $pkgname *.go
+}
+
+package(){
+	cd $pkgname
+    install -Dm 755 $pkgname "$pkgdir/usr/bin/$pkgname"
+	install -Dm 644 -t "$pkgdir/usr/share/doc/$pkgname/" README.md
+	install -Dm 644 LICENSE "$pkgdir/usr/share/licenses/$pkgname/LICENSE"
```

Comment:
You don't have to install GPL3 as it's already available in `/usr/share/licenses/common/`
 
See : https://wiki.archlinux.org/title/PKGBUILD#license and comment in [go pkgbuild template](https://github.com/BlackArch/blackarch-pkgbuilds/blob/master/PKGBUILD-go#L50)